packages/calcite-components/eslint.config.mjs

```diff
@@ -39,6 +39,11 @@ export default tseslint.config(
               message:
                 "Import named functions from commonTests instead of direct module imports, e.g., import { disabled } from 'tests/commonTests'",
             },
+            {
+              group: ["tests/commonTests/browser/*"],
+              message:
+                "Import named functions from commonTests instead of direct module imports, e.g., import { cancelable } from 'tests/commonTests/browser'",
```

Comment:
Can you update the message to mention that it's for browser mode (experimental) tests?